### DISCUSSION TITLE
Style Keyboard Keys and Shortcuts on Web Pages | /en/2023/02/key-buttons/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1474] [Comments]
When writing keyboard shortcuts on web pages, we often use the &lt;code&gt; tag in HTML, or equivalently, a pair of backticks in Markdown. As a result, these keys are styled as code. I have been &hellip;

https://yihui.org/en/2023/02/key-buttons/

**Comment by njtierney** [comment]
Thanks for this, `Yihui` - do you have a recommended place to put 

```
<script src="https://cdn.jsdelivr.net/npm/@xiee/utils/js/key-buttons.min.js" defer></script>
<link href="https://cdn.jsdelivr.net/npm/@xiee/utils/css/key-buttons.min.css" rel="stylesheet">
```

In a hugo blog?

**Reply by yihui** [reply]
Yes, you can put it in any template file that is used for all pages, e.g., https://github.com/rbind/njtierney.com/blob/master/layouts/partials/head_custom.html
